conans/test/command/install_test.py

```diff
@@ -453,3 +453,25 @@ class Pkg(ConanFile):
         self.assertTrue(error)
         self.assertIn("ERROR: Unable to find 'Hello/0.1@lasote/stable' in remotes",
                       client.out)
+
+    def install_argument_order_test(self):
+        # https://github.com/conan-io/conan/issues/2520
+        conanfile = """from conans import ConanFile
+class TestConan(ConanFile):
+    name = "Hello"
+    version = "0.1"
+"""
+        client = TestClient()
+        client.save({"conanfile.py": conanfile})
+        client.run("install . -o boost:shared=True --build=missing")
+        output_0 = client.out
+        client.run("install . -o boost:shared=True --build missing")
+        output_1 = client.out
+        client.run("install -o boost:shared=True . --build missing")
```

Comment:
Test also with 2 --builds, like ``--build missing --build Boost``, with different orders of path